taxi/views.py

```diff
@@ -1,8 +1,13 @@
+from http.client import HTTPResponse
```

Comment:
The import `HTTPResponse` from `http.client` is not used in this file. Consider removing it to clean up the code.